### PR TITLE
i18n ocs day1 and day2 operations

### PR DESCRIPTION
**Pull request by afreen23**
- create storage cluster
- create backing store
- create bucket class
- add capacity
- disk replacement
- create new pool
- edit bucket class
- disk inventory page

Also i18n the lso plugin, since ocs is dependent on lso package

Signed-off-by: Afreen Rahman <afrahman@redhat.com>

fixed issues

### CODE REVIEW

**Comment by afreen23** [comment]
console chnages : [lso ns addition](https://github.com/openshift/console/pull/7406/files#diff-50cfd6287a8fe63d47b05c4b4245840f528ba76950564108cb9893ceed31a7c7) [ webpack chnages](https://github.com/openshift/console/pull/7406/files#diff-a7def86cc36a2178b4ee07b000c53d49e04068db4bc85967c0758578a2f37700)

File: frontend/packages/ceph-storage-plugin/src/components/attached-devices-mode/lso-disk-inventory/disk-replacement-modal.tsx [hunk 1]
```diff
@@ -106,29 +114,31 @@ const DiskReplacementAction = (props: DiskReplacementActionProps) => {
...
             <Link onClick={close} to={DASHBOARD_LINK}>
-              See data resiliency status
+              {t('ceph-storage-plugin~See data resiliency status')}
             </Link>
           </Alert>
         )}
         <p>
-          Are you sure you want to replace <strong>{diskName}</strong> ?
+          <Trans t={t} ns="ceph-storage-plugin">
+            Are you sure you want to replace <strong>{diskName}</strong> ?
```
Comment (rawagner) [inline frontend/packages/ceph-storage-plugin/src/components/attached-devices-mode/lso-disk-inventory/disk-replacement-modal.tsx:134]: I think this needs to be 
```suggestion
            Are you sure you want to replace <strong>{{diskName}}</strong> ?
```

File: frontend/packages/ceph-storage-plugin/src/components/attached-devices-mode/lso-disk-inventory/ocs-status-column.tsx [hunk 2]
```diff
@@ -12,44 +14,57 @@ import {
...
       );
     case Status.Offline:
     case Status.NotResponding:
       return (
-        <PopoverStatus statusBody={<ErrorStatus title={status} />}>
+        <PopoverStatus statusBody={<ErrorStatus title={title} />}>
           <span>
-            Troubleshoot disk <strong>{diskName}</strong>{' '}
+            <Trans t={t} ns="ceph-storage-plugin">
+              Troubleshoot disk <strong>{diskName}</strong>
```
Comment (rawagner) [inline frontend/packages/ceph-storage-plugin/src/components/attached-devices-mode/lso-disk-inventory/ocs-status-column.tsx:53]:
```suggestion
              Troubleshoot disk <strong>{{ diskName }}</strong>
```

File: frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx [hunk 3]
```diff
@@ -107,18 +110,20 @@ export const AddCapacityModal = (props: AddCapacityModalProps) => {
 
   return (
     <form onSubmit={submit} className="modal-content modal-content--no-inner-scroll">
-      <ModalTitle>Add Capacity</ModalTitle>
+      <ModalTitle>{t('ceph-storage-plugin~Add Capacity')}</ModalTitle>
       <ModalBody>
-        Adding capacity for <strong>{getName(ocsConfig)}</strong>, may increase your expenses.
+        <Trans t={t} ns="ceph-storage-plugin">
+          Adding capacity for <strong>{getName(ocsConfig)}</strong>, may increase your expenses.
+        </Trans>
```
Comment (rawagner) [inline frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx:117]:
```suggestion
        <Trans t={t} ns="ceph-storage-plugin" variables={{ name: getName(ocsConfig)}}>
          Adding capacity for <strong>{{name}}</strong>, may increase your expenses.
        </Trans>
```

File: frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/create-sc.tsx [hunk 4]
```diff
@@ -316,12 +320,20 @@ const CreateSC: React.FC<CreateSCProps> = ({ match, hasNoProvSC, mode, lsoNs })
             className="co-alert ocs-install-info-alert"
             variant="info"
             isInline
-            title={!hasNoProvSC ? 'Missing storage class' : 'Internal - Attached devices'}
+            title={
+              !hasNoProvSC
+                ? t('ceph-storage-plugin~Missing storage class')
+                : 'Internal - Attached devices'
```
Comment (rawagner) [inline frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/create-sc.tsx:326]: missing `t`

Comment (afreen23) [reply frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/create-sc.tsx:326]: We are suggested by marketing team to not i18n this the ocs modes  - Internal/Internal-Attached devices/External.
Hence keeping it in plain english

**Pull request merged**